pkg/restore/restore.go

```diff
@@ -1831,3 +1874,81 @@ func (ctx *restoreContext) getSelectedRestoreableItems(resource, targetNamespace
 	}
 	return restorable, warnings, errs
 }
+
+// removeRestoreLabels removes the restore name and the
+// restored backup's name.
+func removeRestoreLabels(obj metav1.Object) {
+	labels := obj.GetLabels()
+
+	if labels == nil {
+		labels = make(map[string]string)
+	}
+
+	labels[velerov1api.BackupNameLabel] = ""
+	labels[velerov1api.RestoreNameLabel] = ""
+
+	obj.SetLabels(labels)
+}
+
+// updates the backup/restore labels
+func (ctx *restoreContext) updateBackupRestoreLabels(fromCluster, fromClusterWithLabels *unstructured.Unstructured, namespace string, resourceClient client.Dynamic) (warnings, errs Result) {
+	patchBytes, err := generatePatch(fromCluster, fromClusterWithLabels)
+	if err != nil {
+		ctx.log.Infof("error generating patch for %s %s: %v", fromCluster.GroupVersionKind().Kind, kube.NamespaceAndName(fromCluster), err)
```

Comment:
Should this be an error log since it is generating an error?

Reply:
Done